apiary.apib

```diff
@@ -1,458 +1,221 @@
 FORMAT: 1A
-# Rails API Template
+HOST: http://rails5-api-base.herokuapp.com
```

Comment:
what things were manually changed in this file by you? in which cases do we need to manually update it?

Reply:
now that I'm thinking. what generates this file?

Reply:
wait, `.apib` means api blueprint. Didn't we switch to OpenAPI? I guess this file needs to be removed now?

Reply:
Yeah, I have to remove this file and update the apiary with the new openapi.yaml once this merges